src/battle_util.c

```diff
@@ -2373,6 +2378,17 @@ u8 DoFieldEndTurnEffects(void)
                         BattleScriptExecute(BattleScript_TailwindEnds);
                         effect++;
                     }
+                    if (IsAbilityOnSide(gActiveBattler, ABILITY_WIND_RIDER))
+                    {
+                        BattleScriptExecute(BattleScript_WindRiderActivatesEnd2);
+                        effect++;
+                    }
+                    if (IsAbilityOnSide(gActiveBattler, ABILITY_WIND_POWER))
```

Comment:
It's already there. If you notice, it's tied to Electromorphosis' effect.

The thing is that Wind Power does have the same property as Wind Rider by which it can activate by Tailwind too :eyes: ...

Should I make it activate inside `TryActivateWindRiderLoop`? *Which I would of course rename.*

Reply:
I just don't see why wind power is in `ENDTURN_TAILWIND` at all. Even if it can be activated by tailwind (which doesn't really make sense from the ability description but I assume there's been in game testing for this), it belongs in the move's effect